pkg/libvmi/config.go

```diff
@@ -28,6 +28,12 @@ func WithSecretDisk(secretName, volumeName string) Option {
 	return WithLabelledSecretDisk(secretName, volumeName, "")
 }
 
+func WithSysprepSecret(secretName, volumeName string) Option {
```

Comment:
Please swap parameters:
```
func WithSysprepSecret(volumeName, secretName string) Option {
```
And do the same for:
- `WithSysprepConfigMap`
- `newSysprepSecretVolume`
- `newSysprepConfigMapVolume`